extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css

```diff
@@ -115,20 +115,31 @@
   white-space: nowrap !important;
 }
 
-.i-amphtml-lbv[gallery-view] .i-amphtml-lbv-gallery {
-  display: flex;
-  flex-wrap: wrap;
-  align-content: flex-start;
+@media screen and (min-width: 1024px) {
```

Comment:
Adding `screen` can be a good idea, but since the rules inside the media query heavily influence the layout of the gallery, it could look broken if the user ever wants to print the gallery view.